### ISSUE TITLE
How to evaluate the model? 

### ISSUE THREAD

**Issue by kazenokizi** [dongdonghy/repulsion-loss-faster-rcnn-pytorch#2]
Hello, thanks for the great work. I am curious about how to evaluate the fine-tuned model using evaluate.ipynb notebook? Specifically, how shall I prepare **path_gt** and **path_dt** files?
